config/environment.rb

```diff
@@ -3,3 +3,14 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+
+ActionMailer::Base.smtp_settings = {
+  :user_name => 'luciankahn',
+  :password => 'puppyparak33t',
```

Comment:
You probably should remove this and load it from the environment. Otherwise anyone with your github address has your sendgrid password.